.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# OpenAPI CLI Contributing Guide
+
+Hi! We're really excited that you are interested in contributing to OpenAPI CLI. Before submitting your contribution though, please make sure to take a moment and read through the following guidelines.
+
+- [OpenAPI CLI Contributing Guide](#openapi-cli-contributing-guide)
+  - [Issue Reporting Guidelines](#issue-reporting-guidelines)
+  - [Pull Request Guidelines](#pull-request-guidelines)
+  - [Development Setup](#development-setup)
+    - [Commonly used NPM scripts](#commonly-used-npm-scripts)
+  - [Project Structure](#project-structure)
+
+## Issue Reporting Guidelines
+
+- Before opening a new issue, try to make sure the same problem or idea hasn't already been reported. You can do that by going to the [Issues page](https://github.com/Redocly/openapi-cli/issues) in the repository and using the filter `is:issue` combined with some keywords relevant to your idea or problem. It helps us notice that more people have the same issue or use-case, and reduces the chance of getting your issue marked as a duplicate. Plus, you can even find some workarounds for your issue in the comments of a previously reported one!
+  
+- The best way to get your bug fixed is to provide a (reduced) test case. This means listing and explaining the steps we should take to try and hit the same problem you're having. It helps us understand in which conditions the issue appears, and gives us a better idea of what may be causing it.
+
+
+## Pull Request Guidelines
+Before submitting a pull request, please make sure the following is done:
+
+1. Fork the repository and create your branch from master.
+2. Run `npm install` in the repository root.
+3. If you’ve fixed a bug or added code that should be tested, don't forget to add tests!
+4. Ensure the test suite passes (`npm test`). Tip: `npm test -- --watch TestName` is helpful in development.
+5. Format your code with prettier (`npm run prettier`).
+
+## Development Setup
+
+You will need [Node.js](http://nodejs.org) at `v12.0.0+`.
+
+After cloning the repo, run:  
```

Comment:
question: should this be forking or cloning? We advise them to fork the repo above, so maybe we should stick to that advice :)

Reply:
Makes sense, will change now.